docs/guides/08-interactivity-events.md

```diff
@@ -0,0 +1,411 @@
+
+Add this to your code to create a layer as usual, and add it to your map:
+```js
+carto.setDefaultAuth({ user: 'cartovl', apiKey: 'default_public' });
+const source = new carto.source.Dataset('populated_places');
+const viz = new carto.Viz();
+const layer = new carto.Layer('Cities', source, viz);
+layer.addTo(map);
+```
+
+Now add a `loaded` listener to the previous layer:
+```js
+layer.on('loaded', () => {
+    console.log('Cities layer has been loaded!');
+});
+```
+> This event could be useful for example to display some kind of loading animation over your map and then hiding it after the layer has loaded. Notice how the name of the event is `loaded`, not `load`.
+
+If you were using several layers, you could also have a single function to handle them all. For this case, **on** & **off** are available at `carto` namespace, expecting a list of layers like this: `carto.on('loaded', [layer1, layer2], () => { console.log('All layers have loaded'); })`.
+
+Regarding to the `updated` event, it can be useful for some other cases, where the layer's viz changes, for example when you are building an animation. See the [Playing with animations guide](/developers/carto-vl/guides/playing-with-animations).
+
+If you check your work now, it should look like this:
+<div class="example-map">
+    <iframe
+        id="guides-interactivity-step-2"
+        src="/developers/carto-vl/examples/maps/guides/interactivity/step-2.html"
+        width="100%"
+        height="500"
+        frameBorder="0">
+    </iframe>
+</div>
+
+
+### Using dynamic variables
+*Variables* are a way to store and reuse expressions, and that can definitively help you when adding interactions to your visualization, so let's practice a bit with them.
+
+#### Variables without properties
+First you are going to add a variable whose value depends solely on the current map extent. Replace your current `const viz = new carto.Viz();`, with this code that grabs the current displayed features using the *String API*:
+```js
+const viz = new carto.Viz(`
+    @currentFeatures: viewportFeatures()
+`);
+```
+
+And then you should add this to handle the updates after you change the map's extent:
+```js
+const displayNumberOfCities = () => {
+    const numberOfFeatures = viz.variables.currentFeatures.value.length;
+    console.log(`Now you can see ${numberOfFeatures} cities`);
+};
+layer.on('updated', displayNumberOfCities);
+```
+> Notice how the variable can be accessed directly from the `carto.Viz` object, inside its `variables` array, without the `@` symbol. Its content is accessible using `.value`, and this is possible cause the expression has no `properties` related to the features themselves.
+
+You can imagine `layer:updated` event as a kind of `layer:viz-updated` event, notifying you whenever something relevant has changed in the viz attached to the layer.
```

Comment:
as a kind of `layer:viz-updated` event --> as a `layer:viz-updated` event

Reply:
This event doesn't exist, so I'll keep something like: _as a "kind of" `layer:viz-updated` event_